core/src/test/java/hudson/ChannelRule.java

```diff
@@ -33,12 +34,12 @@
 
         Future<Channel> f1 = executors.submit(new Callable<Channel>() {
             public Channel call() throws Exception {
-                return new Channel("This side of the channel", executors, p1i, p2o);
+                return new ChannelBuilder("This side of the channel", executors).build(p1i, p2o);
```

Comment:
I'm not sure why. But, I'm guessing that this commit is going to fail. Assuming it does, and the one before it succeeds, I'm going to try dropping it and continuing along w/ the remainder.

Reply:
For reference, this log shows which commits passed/failed: https://github.com/jenkinsci/jenkins/commits/4d894a0bf3929b1fc6712333a7babcdb6a1a995c and 12ae67d295daa59e507dedc9eb1f2e0fd2189a94 indeed failed.

Reply:
It appears to be the lack of `.withMode(Channel.Mode.BINARY)`:

https://github.com/jenkinsci/remoting/blob/01a821470b1657b065beebe421f1dbdd6eb6c0ff/src/main/java/hudson/remoting/Channel.java#L407